src/main/java/org/jenkinsci/plugins/workflow/support/storage/BulkFlowNodeStorage.java

```diff
@@ -149,7 +149,7 @@ public void storeNode(@Nonnull FlowNode n, boolean delayWritingActions) throws I
 
     @Override
     public void storeNode(FlowNode n) throws IOException {
-        storeNode(n, false);
+        flushNode(n);
```

Comment:
Tiny quibble(s) - given that we don't have `@Nonnull FlowNode n` here, it feels like the call should be the other way around. But then, the abstract of `storeNode(FlowNode)` does have `@Nonnull FlowNode n`, so I guess really the answer is to put the annotation here too. Either way it's nitpicking at best. =)